metadata/stream/rationale.yaml

```diff
@@ -157,7 +157,8 @@ architectures:
       kubevirt:
         release: 30.1.2.3
         # ContainerDisk in a container registry
-        image: exampleregistry.io/fcos/fcos@sha256:67a81539946ec0397196c145394553b8e0241acf27b14ae9de43bc56e167f773
+        image: exampleregistry.io/fcos/fcos:stable
+        digest-ref: exampleregistry.io/fcos/fcos@sha256:67a81539946ec0397196c145394553b8e0241acf27b14ae9de43bc56e167f773
```

Comment:
should we add comments here similar to what is done for GCP above about what we recommend versus what we allow.

Also I'm sure we could bikeshed for days on naming here. I'm going to resist because I don't have anything better to offer.

One other really dumb question.., which I suspect the answer is "no". If someone took the container image file from s3 and uploaded it to a different registry (say `newexampleregistry.io`) would the digest be the same over there? `newexampleregistry.io/fcos/fcos@sha256:67a81539946ec0397196c145394553b8e0241acf27b14ae9de43bc56e167f773`?

Reply:
Comments added.

As I understand it, the image digest won't change if the image is uploaded elsewhere.

Reply:
Yep, I think we have to be careful with multi arch image copies , but we should always be able to copy without digest change.

Reply:
The images we generate actually default to OCI format.  Until earlier this year, quay.io for example only supported v2s2 format, and tooling such as `podman push` will auto-convert the image.  However, this necessarily changes the digest.  This is why `podman push --digestfile` (and `skopeo copy --digestfile`) exists - to retrieve the now-canonical pushed image digest.

xref https://github.com/coreos/coreos-assembler/pull/2726